src/api/EpiData.ts

```diff
@@ -9,7 +9,7 @@ import {
   fluViewRegions,
   gftLocations,
   ghtLocations,
-  nidssDenqueLocations,
+  nidssDengueLocations,
```

Comment:
The NIDSS Dengue signal is referred to as "denque" in a few parts of the codebase. Because of this, it is currently broken on https://delphi.cmu.edu/epivis/. This PR also renames all mentions of "denque" to the correct "dengue".

Reply:
Done! https://github.com/cmu-delphi/www-epivis/pull/68

Reply:
Awesome!  Can you merge #68 and then rebase this PR?